pkg/crc/machine/bundle/repository.go

```diff
@@ -89,11 +81,23 @@ func (bundle *CrcBundleInfo) installSymlinkOrCopy(ocBinDir string) error {
 	return os.Symlink(ocInBundle, ocInBinDir)
 }
 
-func Extract(sourcepath string) (*CrcBundleInfo, error) {
-	_, err := extract.Uncompress(sourcepath, constants.MachineCacheDir, true)
-	if err != nil {
+func (r *Repository) Extract(path string) error {
```

Comment:
Wondering if this would be better as `func (repo *Repository)`? I usually have a hard time with one or two letter variable names.

Reply:
done also :)